executor/executor_test.go

```diff
@@ -5623,9 +5623,10 @@ func TestAdmin(t *testing.T) {
 	require.NoError(t, err)
 	row = req.GetRow(0)
 	require.Equal(t, 6, row.Len())
-	txn, err := store.Begin()
+	sess := testkit.NewTestKit(t, store).Session()
+	_, err = sess.Execute(context.Background(), "begin")
 	require.NoError(t, err)
-	ddlInfo, err := ddl.GetDDLInfo(txn)
+	ddlInfo, err := ddl.GetDDLInfo(sess)
 	require.NoError(t, err)
 	require.Equal(t, ddlInfo.SchemaVer, row.GetInt64(0))
 	// TODO: Pass this test.
```

Comment:
remove this comment?

Reply:
looks like it is a very old test. I will keep it here and think maybe we can remove it